app/src/main/java/org/connectbot/di/DatabaseModule.kt

```diff
@@ -39,7 +39,9 @@ object DatabaseModule {
             context,
             ConnectBotDatabase::class.java,
             DATABASE_NAME
-        ).build()
+        )
+            .addMigrations(ConnectBotDatabase.MIGRATION_3_4)
```

Comment:
Strange this has to be added here. It might need to be added to any `TestDatabaseModule` as well.